starctl

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-set -e
+set -x
```

Comment:
So instead of failing eagerly, print out each command to execute? Is removal of former intentional?

Reply:
this was for debugging, reverting the change